account_invoice_constraint_chronology/model/account_invoice.py

```diff
@@ -11,42 +11,36 @@ class AccountInvoice(models.Model):
 
     @api.multi
     def action_move_create(self):
-        previously_validated = self.filtered(lambda inv: inv.move_name)
+        previously_validated = self.filtered(lambda x: x.move_name)
```

Comment:
Perché questa modifica? Giusto per curiosità

Reply:
English :pray:
Local code validator said variable is used in another scope, infact inv. is used below